src/context/video-context.js

```diff
@@ -0,0 +1,213 @@
+import {createContext, useContext, useReducer} from "react";
+
+const VideoContext = createContext();
+export function VideoProvider({children}) {
+  const isLiked = (state, payload) => {
```

Comment:
These functions are meant to be in the reducers and should be able to be dispatched like `dispatch({ type: 'ADD_TO_LIKES', payload })`